doc/src/manual/calling-c-and-fortran-code.md

```diff
@@ -657,7 +657,7 @@ For translating a C argument list to Julia:
       * `Ref{Any}`
       * argument list must be a valid Julia object (or `C_NULL`)
       * cannot be used for an output parameter, unless the user is able to
```

Comment:
This sentence just seems awkward as a whole. Maybe drop the `manage to` as well? Seems grammatically redundant with the previous `is able to`.

Reply:
Yes I think that reads better.